package.json

```diff
@@ -41,13 +41,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@types/jest": "^29.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.38.0",
-    "@typescript-eslint/parser": "^5.38.0",
-    "eslint": "^8.24.0",
-    "eslint-config-airbnb": "^19.0.4",
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
+    "@typescript-eslint/parser": "^5.38.1",
     "eslint-config-airbnb-typescript": "^17.0.0",
-    "jest": "^29.0.3",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint": "^8.24.0",
     "jest-environment-jsdom": "^29.0.3",
+    "jest": "^29.0.3",
+    "react-dom": "^18.2.0",
+    "servatron": "^2.3.2",
```

Comment:
Do I need this? Was this part of a bad merge conflict?

Reply:
Yeah this looks like a merge conflict thing. It's not needed. Neither is `react-dom`.